### DISCUSSION TITLE
How to set proxy and add existing jupyter server by command or condig or setting file?

### DISCUSSION THREAD

**Discussion by graczhual** [microsoft/vscode-jupyter#16082] [Questions and Answers]
First, I would like to ask a basic question. When we use Jupyter on VS Code, is it only possible to preserve the session after closing the file by adding an existing Jupyter server, and not sufficient just by adding a local kernel?
Secondly, by setting up a proxy, I am now able to successfully add an existing Jupyter server. I would like to know if this process can be accomplished through a command or by configuring config or setting files, as it involves some tokens and I hope to complete it in a more concealed manner.

**Comment by DonJayamanne** [comment]
Closing as a duplicate of https://github.com/microsoft/vscode-jupyter/issues/3998
Unfortunately we do not have an option to start jupyter servers as requested.
Based on your comments, I believe the above issue will address your requriements.
